workflows/pipe-common/pipeline/hpc/param.py

```diff
@@ -249,6 +249,12 @@ def __init__(self):
         self.event_ttl = GridEngineParameter(
             name='CP_CAP_AUTOSCALE_EVENT_TTL', type=PARAM_INT, default=3 * 60 * 60,
             help='Specifies event ttl in seconds after which an event is removed.')
+        self.sge_selected = GridEngineParameter(
+            name='CP_CAP_SGE', type=PARAM_BOOL, default=True,
+            help='Defines if SGE selected as grid engine.')
+        self.slurm_selected = GridEngineParameter(
+            name='CP_CAP_SLURM', type=PARAM_BOOL, default=False,
+            help='Defines if Slurm selected as grid engine.')
```

Comment:
Let's move these parameters to `GridEngineQueueParameters` class instead because they describe a queue that should be autoscaled.

Reply:
From my point of view its describing general settings of autoscaling rather then queue, don't you think?

Reply:
In practice these two settings specify a type of a grid engine queue. And I believe that it is the characteristic of a queue the same way as queue name is.